contract/README.md

```diff
@@ -80,22 +88,71 @@ Each environment should have their separate keypair - this is important.
 
 See `/web-server/README.md` for more information.
 
----
+**As this is an OpenZeppelin Upgradeable Contract**, there are two types of deploys now:
+
+-   an initial deploy (only do once to setup initial 3 contracts), and
+-   an upgrade deploy on each subsequent release
 
 **Localhost (hardhat)**
 
 ```shell
-yarn deploy
+yarn compile && yarn deploy
+```
+
+or
+
+```shell
+# make sure UPGRADEABLE_PROXY_ADDRESS .env has a value that points to OpenZeppelin Proxy contract.
+yarn compile && yarn upgrade
```

Comment:
```suggestion
yarn compile && yarn run upgrade
```

Unfortunately `yarn upgrade` clashes with the yarn command for upgrading package dependencies. I was wondering how come it took so long when I ran the command 😄

Reply:
Nice catch, thanks!